sklego/preprocessing/patsytransformer.py

```diff
@@ -1,9 +1,21 @@
+try:
+    import patsy
+except ImportError:
+    from sklego.notinstalled import NotInstalledPackage
+
+    patsy = NotInstalledPackage("patsy")
+
 import numpy as np
-from patsy import dmatrix, build_design_matrices, PatsyError
+from deprecated import deprecated
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.utils.validation import check_is_fitted
 
 
+@deprecated(
+    version="0.6.17",
+    reason="Please use `sklego.preprocessing.FormulaicTransformer` instead. "
+    "This object will be removed from the preprocessing submodule in version 0.9.0.",
```

Comment:
Totally made up version number in which to deprecate 😂

Reply:
No worries :) That said ... I'd be fine with 0.8.0.